thecampuscookbook/thecampuscookbook/models.py

```diff
@@ -0,0 +1,54 @@
+from django.utils.timezone import timezone
```

Comment:
The models defined in this file should actually be placed in the correct `models.py` file for each application ([source](https://stackoverflow.com/questions/17553240/where-should-i-place-models-py-in-django)).  
If you checkout `main`, you'll notice multiple application directories, each with the `model.py` file.  
I'm not sure if this is the correct place to keep this `models.py` file because this `models.py` might not be accessible to the migrate command in the `manage.py` script.